docs/resources/aws_efs_mount_targets.md

```diff
@@ -0,0 +1,78 @@
+---
+title: About the aws_efs_mount_targets Resource
+platform: aws
+---
+
+# aws\_efs\_mount\_targets
+
+Use the `aws_efs_mount_targets` InSpec audit resource to test properties of a plural EFS Mount Targets.
+
+The AWS::EFS::MountTarget resource is an Amazon EFS resource that creates a mount target for an EFS file system. You can then mount the file system on Amazon EC2 instances or other resources by using the mount target.
+
+## Syntax
+
+Ensure that a file system exists.
+    describe aws_efs_mount_targets(file_system_id: "fs-01234567") do
+      it { should exist }
+    end
+
+## Parameters
+
+`file_system_id` _(required)_
```

Comment:
Is it`file_system_id` or `file_system_ids`. See the properties list below.

Reply:
It should be file_system_ids. I have corrected it and pushed the changes.